Account.java

```diff
@@ -0,0 +1,86 @@
+package accounting;
+
+import agents.Agent;
+import actions.Action;
+import contracts.Contract;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+public class Account {
+
+    private Account(String name, AccountType accountType, Double startingBalance) {
+        this.name = name;
+        this.accountType = accountType;
+        this.balance = startingBalance;
+        this.contractClass = null;
+        this.contracts = new HashSet<>();
+    }
+
+    Account(String name, AccountType accountType) {
+        this(name,accountType,0.0);
+    }
+
+    private double balance;
+
+//    private Collateral collateralType;
```

Comment:
remove commented variables, unless it is a discussion point whether or not to add them.